ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -249,16 +250,24 @@ internal class MockApplicationContext : ApplicationContext {
             else -> error("Unknown string resource $resId")
         }
     }
+
+    override fun getApplicationPackageName() = "com.revenuecat.mockpaywall"
 }
 
 internal class MockViewModel(
     private val mode: PaywallViewMode = PaywallViewMode.default,
-    private val offering: Offering,
+    offering: Offering,
 ) : ViewModel(), PaywallViewModel {
     override val state: StateFlow<PaywallViewState>
         get() = _state.asStateFlow()
-    private val _state =
-        MutableStateFlow(offering.toPaywallViewState(VariableDataProvider(MockApplicationContext()), mode))
+    private val _state = MutableStateFlow(
+        offering.toPaywallViewState(
+            VariableDataProvider(MockApplicationContext()),
+            mode,
+            TestData.template2,
+            PaywallTemplate.TEMPLATE_2,
```

Comment:
This is no longer getting the paywall from `offering`. Since this is just for test data, maybe do:
```suggestion
            offering.paywall!!,
            PaywallTemplate.fromId(offering.paywall!!)!!,
```